apps/docs/src/app/platform/component-docs/platform-smart-filter-bar/platform-smart-filter-bar-examples/platform-smart-filter-bar-basic-example.component.html

```diff
@@ -11,7 +11,7 @@ <h4 fd-title>Line Items</h4>
 >
     <fdp-column
         fdp-smart-filter-bar-field-definition
-        [filterable]="true"
+        [smartFilterBarFilterable]="true"
```

Comment:
Almost everywhere it's set to true, why we would not make it the default value then?

Reply:
They are enabled in our examples to show the variety of the filters. Other users might not want to do that